### DISCUSSION TITLE
Error while copying model using model.copy()

### DISCUSSION THREAD

**Discussion by anubhavdas0907** [micom-dev/micom#69] [General]
Hello Christian,

I was trying to copy a community model to a different variable, but I get an error.
I want to manipulate a model, without changing the original one.

Following are the details.

```
from micom import load_pickle
model = load_pickle("ERR1883210.pickle")
model_1 = model.copy()
```
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/data/Conda_base/envs/MyConda/lib/python3.7/site-packages/cobra/core/model.py", line 321, in copy
    new = self.__class__()
TypeError: __init__() missing 1 required positional argument: 'taxonomy'
```

Can you please suggest, what's wrong in this code, and what can be the solution?

Regards
Anubhav

**Comment by cdiener** [comment]
This is a bug, so I converted it to an issue and mentioned you there :sweat_smile:
